aiida_vasp/workchains/relax.py

```diff
@@ -40,6 +40,13 @@ def define(cls, spec):
                    help="""
             If True, perform relaxation.
             """)
+        spec.input('relax.keep_magnetization',
+                   valid_type=get_data_class('bool'),
+                   required=False,
+                   default=lambda: get_data_node('bool', True),
+                   help="""
+                   If True, try to keep the site magnetization from the previous calculation.
```

Comment:
Do we try? I think we keep it, right?

Reply:
Okey, it raises if something is wrong. This is fine.

Reply:
If one does not parse `site_magnetization` or don't have `lorbit` set then it will not work. No warnings are given in thoses cases - hence the *try*.